src/use-swr-suspense.ts

```diff
@@ -0,0 +1,46 @@
+// TODO: add documentation
+
+import useSWR from './use-swr'
+
+const suspenseGroup = {
+  promises: [],
+  started: false
+}
```

Comment:
What would happen if I use `useSWRSuspense` or manually used `_internal_useSWRSuspenseStart` and `_internal_useSWRSuspenseEnd` in two component at the same time? Couldn't this cause some kind of race condition since both will modify this object at the same time?

Reply:
It will be fine because inside `render`,

```js
useSWRSuspenseStart()
const { data: user } = useSWR('/api/user')
const { data: movies } = useSWR('/api/movies')
...
useSWRSuspenseEnd()    
```

This entire code block is atomic (synchronous). No other JavaScript code can be executed between `useSWRSuspenseStart()` and `useSWRSuspenseEnd()`.

Reply:
And this is still highly experimental. Currently I don't see anyone else in the community using this pattern. It needs more discussions and we are not planning to land it very soon.